assets/js/admin/course-edit.js

```diff
@@ -83,3 +84,19 @@ registerPlugin( 'sensei-course-video-progression-plugin', {
 	render: CourseVideoSidebar,
 	icon: null,
 } );
+
+/**
+ * Filters the course pricing sidebar toggle.
+ *
+ * @since 4.1.0
+ *
+ * @hook  senseiCoursePricingHide
+ * @param {boolean} $hide_course_pricing
+ * @return {boolean}
```

Comment:
It's nice to have descriptions on both the `params` and the `return`. Also, you could use camelCase for the param variable and remove the dollar sign.

Reply:
Good catch: https://github.com/Automattic/sensei/pull/4810/files#diff-46b5b32718313cd62a309f3e5bb5b44af7fd7d418aa97369ac2e33b8204b7d35R94

Reply:
By adding descriptions I mean adding short sentences describing each row. Here's an [example](https://github.com/Automattic/sensei/blob/master/assets/blocks/quiz/answer-blocks/index.js#L161-L173).

Reply:
Added here: [Link](https://github.com/Automattic/sensei/pull/4810/files#diff-46b5b32718313cd62a309f3e5bb5b44af7fd7d418aa97369ac2e33b8204b7d35R81)